src/components/shared/Response/domHandlers.tsx

```
@@ -0,0 +1,196 @@
+  showSources?: boolean;
+  linkColor?: string;
+  latexExpressions?: Map<string, LaTeXExpression>;
+}
+
+// Constants
+const NUMBER_REFERENCE_REGEX = /\[\d+(,\s*\d+)*\]/g;
+
+export class DomNodeHandlers {
+  public handleCodeBlock(
+    domNode: DomNode,
+    createParserOptions: Function,
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)